server/views/pages/index.njk

```diff
@@ -1,33 +1,61 @@
 {% extends "../partials/layout.njk" %}
+{% from "govuk/components/table/macro.njk" import govukTable %}
+{%- from "moj/components/search/macro.njk" import mojSearch -%}
+{% from "govuk/components/button/macro.njk" import govukButton %}
 
 {% set pageTitle = applicationName + " - Home" %}
 
 {% block content %}
 
     <div class="govuk-grid-row">
-        <div class="govuk-grid-column-two-thirds">
-            <h1 class="govuk-heading-l">Manage a recall for an offender on licence</h1>
-
-            <p class="govuk-body">As a community offender manager, use this service to:</p>
-
-            <ul class="govuk-list govuk-list--bullet">
-                <li>initiate a recall request for an offender on licence</li>
-                <li>notify Public Protection Casework Section (PPCS) of the recall request</li>
-                <li>track progress of a recall</li>
-            </ul>
-            <p class="govuk-body">By using it, you no longer need to fill in the Part A report.</p>
-
-
-            <a href="/recall/search" role="button" draggable="false" class="govuk-button govuk-!-margin-top-2 govuk-!-margin-bottom-8 govuk-button--start" data-module="govuk-button" data-qa="start-recall">
-                Start now
-                <svg class="govuk-button__start-icon" xmlns="http://www.w3.org/2000/svg" width="17.5" height="19" viewBox="0 0 33 40" aria-hidden="true" focusable="false">
-                    <path fill="currentColor" d="M0 0h13l20 20-20 20H0l20-20z" />
-                </svg>
-            </a>
-
+        <div class="govuk-grid-column-full">
+            <h1 class="govuk-heading-l">
+                Find an offender
+            </h1>
+
+            <form id="search" action="" method="post" class="govuk-!-padding-5 govuk-!-margin-bottom-9 search-panel">
+                <input type="hidden" name="_csrf" value="{{ csrfToken }}"/>
+                <div class="moj-search">
+                    <div class="govuk-form-group">
+                        <label class="govuk-label moj-search__label govuk-!-font-weight-bold" for="search">
+                            Enter a NOMIS number
+                        </label>
+                        <input class="govuk-input moj-search__input " id="search-field" name="nomisNumber" type="search"
+                               aria-describedby="search-hint" data-qa="search-field">
+                    </div>
+                    <button type="submit" class="govuk-button moj-search__button " data-module="govuk-button" data-qa="search">
+                        Search
+                    </button>
+                </div>
+            </form>
+
+            {% if prisoners %}
+                <h2 class="govuk-heading-m">
+                    {{ prisoners.length }} people found
+                </h2>
+                <table class="govuk-table govuk-radios govuk-radios--small">
```

Comment:
assume we can remove `govuk-radios govuk-radios--small`

Reply:
Good spot, I'll fix in the next PR